Formula/boost-python.rb

```diff
@@ -1,9 +1,9 @@
 class BoostPython < Formula
```

Comment:
Is this formula still needed anywhere? Otherwise it should probably be deprecated given the state of python 2

Reply:
I was thinking that too, but then got distracted by the other formula. Can we just remove it? I don't know if I can get it to build.

Fine, we probably can't, it has ~30,000 installs in the past year

Reply:
Oh, as for being needed anywhere: no, no formulae depend on it.

Reply:
It should probably be deprecated from January 1, 2020 then.